libs/server/Resp/RespServerSession.cs

```diff
@@ -249,6 +254,10 @@ public RespServerSession(
                 if (this.networkSender.GetMaxSizeSettings?.MaxOutputSize < sizeof(int))
                     this.networkSender.GetMaxSizeSettings.MaxOutputSize = sizeof(int);
             }
+
+            // grab stream manager from storeWrapper
+            this.streamManager = storeWrapper.streamManager;
```

Comment:
We need a flag to enable streams `--streams` and leave it disabled by default. Similar to `--lua` which enables Lua. If not enabled, we should not incur the overhead of creating stream manager or session stream cache at all.